### PR TITLE
Diving Deep into Lazy Vals

### PR DESCRIPTION
**Pull request by chaitanyawaikar**

### CODE REVIEW

File: core-scala/src/test/scala/com/baeldung/scala/LazyValTest.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import org.scalatest.FunSuite
+import org.scalatest.Matchers._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Future, _}
+import com.baeldung.scala
+
+class LazyValTest extends FunSuite {
```
Comment (KevinGilmore) [inline core-scala/src/test/scala/com/baeldung/scala/LazyValTest.scala]: The file (and class name) should end in UnitTest

File: core-scala/src/main/scala/com/baeldung/scala/LazyVal.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package com.baeldung.scala
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent._
+import scala.concurrent.duration._
+
+class Member {
```
Comment (KevinGilmore) [inline core-scala/src/main/scala/com/baeldung/scala/LazyVal.scala]: Would it be better to rename the class to LazyVal?

Comment (chaitanyawaikar) [reply core-scala/src/main/scala/com/baeldung/scala/LazyVal.scala]: It's a good suggestion. I have renamed it to LazyVal

File: core-scala/src/test/scala/com/baeldung/scala/LazyValTest.scala (hunk 1 above)
Comment (chaitanyawaikar) [reply core-scala/src/test/scala/com/baeldung/scala/LazyValTest.scala]: The name of the file has been changed.

**Pull request merged**